### PR TITLE
[PIP-16] Add boilerplate

### PR DESCRIPTION
**Pull request by EijiTomonari**
## What does this Pull Request Do?
- Initialize the plugin work tree with a code boilerplate

## Where should the reviewer start?
- Read the documentation from the original [repo](https://github.com/TukuToi/better-wp-plugin-boilerplate)
- Read the new files

## How is this tested?
- Go to `wp-admin/plugins.php` and activate the plugin
- Check if the message is successful

### CODE REVIEW

File: src/pipe-web-monetization.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+ * admin area. This file also includes all of the dependencies used by the plugin,
+ * registers the activation and deactivation functions, and defines a function
+ * that starts the plugin.
+ *
+ * @link              https://example.com
+ * @since             1.0.0
+ * @package           Pipe_Web_Monetization
+ *
+ * @wordpress-plugin
+ * Plugin Name:       Pipe Web Monetiation
```
Comment (pivagabriel) [inline src/pipe-web-monetization.php]: Change 'Monetiation' to 'Monetization'

**Review by pivagabriel — CHANGES_REQUESTED** [review]
Fix plugin name

File: src/admin/css/pipe-web-monetization-admin.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/**
+ * All of the CSS for your admin-specific functionality should be
+ * included in this file.
+ */
```
Comment (eusouoandrey) [inline src/admin/css/pipe-web-monetization-admin.css]: Missing an empty line at the end of this file

File: src/pipe-web-monetization.php (hunk 1 above)
Comment (EijiTomonari) [reply src/pipe-web-monetization.php]: Thx

File: src/public/css/pipe-web-monetization.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+/**
+ * All of the CSS for your public-facing functionality should be
+ * included in this file.
+ */
```
Comment (eusouoandrey) [inline src/public/css/pipe-web-monetization.css]: Missing empty line

**Review by eusouoandrey — COMMENTED** [review]
Just some comments. Good job, bro!

File: src/admin/css/pipe-web-monetization-admin.css (hunk 2 above)
Comment (EijiTomonari) [reply src/admin/css/pipe-web-monetization-admin.css]: Nice

**Review by pivagabriel — APPROVED** [review]

**Pull request merged**